### PR TITLE
Add-volta-settings

### PR DESCRIPTION
**Pull request by gyfchong**
## Why
Volta good, NVM bad
https://docs.volta.sh/guide/

## What
Added a volta config to package json to let volta know what to use

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -141,5 +141,9 @@
     "turbo": "^1.11.3",
     "typescript": "^5.3.3",
     "webpack": "^5.89.0"
+  },
+  "volta": {
```
Comment (HeartSquared) [inline package.json:145]: Do you need to remove `.nvmrc` then?

Comment (gyfchong) [reply package.json:145]: Nah, they can work independently in the same repo. Also I think NVM is more popular at CA.

Comment (HeartSquared) [reply package.json:145]: Should we even be committing this change then if no one else is meant to use it 😛

Comment (gyfchong) [reply package.json:145]: I don't think there's any explicit standard that says we must use NVM... Just so happens to be the most common one found.

**Review by HeartSquared — APPROVED** [review]
Can't hurt, right?

**Pull request merged**